src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -190,4 +191,25 @@ private function decideUnanimous(TokenInterface $token, array $attributes, $obje
 
         return $this->allowIfAllAbstainDecisions;
     }
+
+    /**
+     * TokenInterface vote proxy method.
+     *
+     * Acts as a BC layer when the VoterInterface is not implemented on the voter.
+     */
```

Comment:
I could make the method itself deprecated with a full-fledged warning, even though it's private. Would make it easier to detect.